src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/util/Reviewable.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.util
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Licence.Companion.SYSTEM_USER
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Staff
+import java.time.LocalDateTime
+
+interface Reviewable {
+  var dateLastUpdated: LocalDateTime?
+  var updatedBy: Staff?
+  var updatedByUsername: String?
+  var reviewDate: LocalDateTime?
+
+  fun markAsReviewed(staff: Staff?) {
+    reviewDate = LocalDateTime.now()
+    dateLastUpdated = LocalDateTime.now()
+    updatedByUsername = staff?.username ?: SYSTEM_USER
+    updatedBy = staff ?: this.updatedBy
+  }
```

Comment:
I think this will be okay,.. was concerned about an issue I had with mixins and JPA,... but becuase updatedBy is in Licence I think this will be okay! Still confused with the name because all licences are  Reviewable, but I dont know what a good name will be as I dont know the context!